regexes/client/browsers.yml

```
@@ -515,6 +515,27 @@
   name: 'Baidu Spark'
   version: '$1'
 
+# Yandex Browser (alpha)
+- regex: 'YaBrowser(?:/(\d+[\.\d]*)) \(alpha\)?'
+  name: 'Yandex Browser (alpha)'
+  version: '$1'
+  engine:
+    default: 'Blink'
+
+# Yandex Browser (beta)
+- regex: 'YaBrowser(?:/(\d+[\.\d]*)) \(beta\)?'
+  name: 'Yandex Browser (beta)'
+  version: '$1'
+  engine:
+    default: 'Blink'
+
+# Yandex Browser Lite
+- regex: 'YaBrowser(?:/(\d+[\.\d]*)) \(lite\)?'
```

Is the last `?` on the right position?

---

Yes, is based on the regex for Yandex Browser.

---

```suggestion
- regex: 'YaBrowser(?:/(\d+[\.\d]*)) \(lite\)'
```

The `?` in the end is indeed kind of useless. Or might it be possible the the user agent only contains `(lite` (without the closing `)`)